cmake/templates/OpenVINOConfig.cmake.in

```diff
@@ -11,7 +11,7 @@
 #
 #  * `Runtime`: OpenVINO C++ and C Core & Inference Runtime, frontend common
 #  * `ONNX`: OpenVINO ONNX frontend
-#  * `PaddlePaddle`: OpenVINO PaddlePaddle frontend
+#  * `paddle`: OpenVINO paddle frontend
```

Comment:
I suppose here we can use a capital letter

Reply:
fixed